TrelloAutotest/Pages/TrelloBoardPage.cs

```diff
@@ -1,45 +1,56 @@
-﻿using OpenQA.Selenium;
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
 using System.Linq;
 using TrelloAutotest.Messages;
+using TrelloAutotest.Selectors;
 using TrelloAutotests.Pages;
 
 namespace TrelloAutotest.Pages
 {
-    using Selectors = Selectors.BoardSelectors;
-    using BaseSelectors = Selectors.BaseSelectors;
-
     public class TrelloBoardPage : BasePage
     {
-        public TrelloBoardPage() => Wait.Until(d => driver.FindElements(Selectors.BoardHeader).Any());
-
         public void OpenBoardMenu() => driver.FindElements(By.TagName("a")).First(x => x.Text.Contains(MessageText.ShowMenu)).Click();
-        
-        public bool IsLoginCorrect() => driver.FindElements(Selectors.BoardHeader).Any();
 
-        public bool IsUrlCorrect(string BoardName)
+       
+
+        public TrelloBoardPage IsUrlCorrect(string BoardName)
         {
-            Wait.Until(d => driver.FindElements(Selectors.BoardViewButton).Any());
+            Wait.Until(d => driver.FindElements(BoardSelectors.BoardViewButton).Any());
 
             var SplitName = BoardName.Contains(' ') ? BoardName.ToLower().Replace(' ', '-').TrimEnd('-') : BoardName.ToLower();
```

Comment:
**Parameters** and **variables** should follow camelCase. The name of this var suggests it is the method because it's starting with a verb.

Reply:
it could be written in a shorter way:
`var SplitName = (BoardName.Contains(' ') ? BoardName.Replace(' ', '-').TrimEnd('-') : BoardName).ToLower()`